config/packages/surfnet_saml.yaml

```diff
@@ -17,3 +18,16 @@ surfnet_saml:
       entity_id: "%saml_remote_idp_entity_id%"
       sso_url: "%saml_remote_idp_sso_url%"
       certificate: "%saml_remote_idp_certificate%"
+
+when@test:
+  surfnet_saml:
+    hosted:
+      service_provider:
+        public_key: /vagrant/deploy/tests/behat/fixtures/test_public_key.crt
+        private_key: /vagrant/deploy/tests/behat/fixtures/test_private_key.key
```

Comment:
This can't be right. The test env does not utilize the vagrant files anymore.

Reply:
Tell me what is has to be then